doc/content/microreactors/gcmr/GCMR_results2.md

```diff
@@ -0,0 +1,53 @@
+## Results - Dynamic Multiphysics simulation of RIA
```

Comment:
name of the file `GCMR_result2.mg` is not very explicit. I recommend instead: `GCMR_result_RIA.mg` (and rename the other one `GCMR_result_FlowBlockage.mg`